ballerina/Ballerina.toml

```diff
@@ -18,3 +18,11 @@ path = "../native/build/libs/xslt-native-2.3.0-SNAPSHOT.jar"
 [[platform.java11.dependency]]
 path = "./lib/jakarta.activation-1.2.2.jar"
 version = "1.2.2"
```

Comment:
Do we need jakarta dependency?

Reply:
Yes, it is required dependancy for the code. Otherwise we get `ClassNotFoundException` in runtime